index.html

```diff
@@ -88,29 +85,19 @@
             Intel CPUs, with AMD and Arm support in developer preview.
           </p>
           <p>
-            Firecracker is integrated with
+            Firecracker is used by/integrated with (in alphabetical order):
+            <a href="https://appfleet.com/">appfleet</a>, containerd via
+            <a href="https://github.com/firecracker-microvm/firecracker-containerd">firecracker-containerd</a>,
+            <a href="https://fly.io">Fly.io</a>,
             <a
               href="https://github.com/kata-containers/documentation/wiki/Initial-release-of-Kata-Containers-with-Firecracker-support"
-            >
-              Kata Containers
-            </a>
-            ,
-            <a href="https://opennebula.io/firecracker/">Open Nebula</a>
-            ,
-            <a href="https://github.com/solo-io/unik">UniK</a>
-            ,
-            <a href="https://www.weave.works/oss/firekube/">Weave FireKube</a>
-            (via
-            <a href="https://github.com/weaveworks/ignite">Weave Ignite</a>
-            ), and containerd via
-            <a href="https://github.com/firecracker-microvm/firecracker-containerd">firecracker-containerd</a>
-            It's also used by
-            <a href="https://fly.io/">fly.io</a>
-            . Firecracker can run Linux and
-            <a href="http://blog.osv.io/blog/2019/04/19/making-OSv-run-on-firecraker">OSv</a>
-            guests. Our latest roadmap can be found
-            <a href="https://github.com/firecracker-microvm/firecracker/projects/13">here</a>
-            .
+              >Kata Containers</a
```

Comment:
Let's break the line by having "Kata Containers" alone, and the `<a ...>` and `</a>` tags on their own lines too.